src/PKSim.Presentation/Presenters/PopulationAnalyses/PopulationAnalysisAvailablePKParametersPresenter.cs

```diff
@@ -84,8 +91,9 @@ public void QuantityPKParameterSelected(QuantityPKParameterDTO quantityPKParamet
       private IReadOnlyCollection<QuantityPKParameterDTO> availablePKParametersFor(IQuantity quantity)
       {
          var quantityDTO = _quantitySelectionDTOMapper.MapFrom(quantity);
+         var quantityDisplayPth = _quantityDisplayPathMapper.DisplayPathAsStringFor(quantity, addSimulationName: false);
```

Comment:
This will be a unified path were organism will be removed and simulation will not be visible

Reply:
problem also for PK Parameters